src/common/roll_renderer.js

```diff
@@ -410,13 +410,16 @@ class Beyond20RollRenderer {
             let d20_modifier = request.reliableTalent ? "min10" : "";
             if (request.silverTonge && (request.skill === "Deception" || request.skill === "Persuasion"))
                 d20_modifier = "min10";
+            if (request.indomitableMight && request.skill === "Athletics")
```

Comment:
You don't need to check for the skill, the request won't have indomitableMight if it's not athletics.